Test domains/Banking.Domain/CheckingAccount/Commands/CloseCheckingAccount.cs

```diff
@@ -1,20 +1,39 @@
 // Copyright (c) Microsoft. All rights reserved. 
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Threading.Tasks;
 using Its.Validation;
 using Its.Validation.Configuration;
 using Microsoft.Its.Domain;
 
 namespace Test.Domain.Banking
 {
-    public class CloseCheckingAccount : Command<CheckingAccount>
+    public partial class CheckingAccount
     {
-        public override IValidationRule<CheckingAccount> Validator
+        public class CloseCheckingAccount : Command<CheckingAccount>
```

Comment:
I recommend against nesting commands inside of aggregate classes as it adds the possibility of modifying private state on the aggregate. If this happens it would not get persisted. All state persistence has to happen via events, so this coding convention is meant to avoid that gotcha.